Neos.Neos/Classes/Domain/Service/SiteImportService.php

```diff
@@ -78,7 +83,8 @@ public function importFromPath(ContentRepositoryId $contentRepositoryId, string
             'Create Neos sites' => new SiteCreationProcessor($this->siteRepository, $this->domainRepository, $this->persistenceManager),
             'Import events' => $this->contentRepositoryRegistry->buildService($contentRepositoryId, new EventStoreImportProcessorFactory(WorkspaceName::forLive(), keepEventIds: true)),
             'Import assets' => new AssetRepositoryImportProcessor($this->assetRepository, $this->resourceRepository, $this->resourceManager, $this->persistenceManager),
-            'Catchup all projections' => new ProjectionCatchupProcessor($this->contentRepositoryRegistry->buildService($contentRepositoryId, new SubscriptionServiceFactory())),
+            // todo we do a replay here even though it will redo the live workspace creation. But otherwise the catchup hooks are not skipped because it seems like a regular catchup
+            'Catchup all projections' => new ProjectionReplayProcessor($contentRepositoryMaintainer),
```

Comment:
we do a replay here even though it will redo the live workspace creation from above. But otherwise the catchup hooks are not skipped because it seems like a regular catchup. Is this fine?

But we originally said https://github.com/neos/neos-development-collection/pull/5307#discussion_r1812734052

> A replay should not be required.
> If it is, we'll have to investigate. But the plan is to publish events just like the CR would do.
> If the catchups are skipped, we could invoke those. But a full replay would be an overkill

Reply:
this would be an overhead if we want to support import into other root workspaces as previous events would be replayed as well